tests/cpp-tests/Classes/ExtensionsTest/NetworkTest/SocketIOTest.cpp

```diff
@@ -195,7 +195,7 @@ void SocketIOTest::closedSocketAction(network::SIOClient* client)
 void SocketIOTest::onMenuSIOClientClicked(cocos2d::Ref *sender)
 {
 	//create a client by using this static method, url does not need to contain the protocol
-	_sioClient = SocketIO::connect("ws://dev.channon.us:3010", *this);
+	_sioClient = SocketIO::connect("ws://localhost:3010", *this);
```

Comment:
If we change the test case to use `localhost`, it means we must have a socket io server locally, it's not very friendly for testing. Is this necessary?